Code/GraphMol/MolEnumerator/MolEnumerator.h

```diff
@@ -237,6 +240,55 @@ class RDKIT_MOLENUMERATOR_EXPORT RepeatUnitOp : public MolEnumeratorOp {
   void initFromMol();
 };
 
+using stereo_flipper_t = std::shared_ptr<StereoFlipper>;
+
+struct RDKIT_MOLENUMERATOR_EXPORT StereoEnumerationOptions {
+  bool tryEmbedding = false;
+  bool onlyUnassigned = true;
+  bool onlyStereoGroups = false;
+  bool unique = true;
+  unsigned int maxIsomers = 1024;
+};
+
+RDKIT_MOLENUMERATOR_EXPORT unsigned int get_stereoisomer_count(
```

Comment:
```suggestion
RDKIT_MOLENUMERATOR_EXPORT unsigned int getStereoisomerCount(
```
Same suggestion for ay other snakeCase